a10_octavia/db/repositories.py

```diff
@@ -520,6 +520,23 @@ def get_lbs_on_thunder_by_subnet(self, session, loadbalancer_id, subnet_id):
             return None
         return model.to_data_model()
 
+    def get_pending_lbs_to_be_deleted(self, session, cleanup_interval):
+        lb_list = []
+        time_interval = datetime.datetime.utcnow() - datetime.timedelta(
+                seconds=CONF.a10_house_keeping.resource_cleanup_interval)
```

Comment:
seems we already pass CONF.a10_house_keeping.resource_cleanup_interval in as cleanup_interval parameter. So, maybe just `seconds=cleanup_interval`

Reply:
yes, changed it to `seconds=cleanup_interval`